### ISSUE TITLE
VSCode Tach output: ModuleNotFoundError: No module named 'tach.check'

### ISSUE THREAD

**Issue by algorithmx** [gauge-sh/tach-vscode#8]
Installed on VSCode
Run the command Tach: Restart server

Error message:
```
2024-10-08 21:51:23.042 [info] [Error - 9:51:23 PM] Server initialization failed.
2024-10-08 21:51:23.043 [info]   Message: Pending response rejected since connection got disposed
  Code: -32097 
2024-10-08 21:51:23.043 [info] [Info  - 9:51:23 PM] Connection to server got closed. Server will restart.
2024-10-08 21:51:23.043 [info] true
2024-10-08 21:51:23.043 [info] [Error - 9:51:23 PM] Tach client: couldn't create connection to server.
2024-10-08 21:51:23.043 [info]   Message: Pending response rejected since connection got disposed
  Code: -32097 
2024-10-08 21:51:23.043 [info] [Error - 9:51:23 PM] Restarting server failed
2024-10-08 21:51:23.043 [info]   Message: Pending response rejected since connection got disposed
  Code: -32097 
2024-10-08 21:51:23.244 [info] Traceback (most recent call last):
  File "/home/dabajabaza/.vscode/extensions/gauge.tach-0.10.3-linux-x64/bundled/tool/lsp_server.py", line 47, in <module>
    from tach_util import run_tach_check  # noqa: E402
  File "/home/dabajabaza/.vscode/extensions/gauge.tach-0.10.3-linux-x64/bundled/tool/tach_util.py", line 3, in <module>
    from tach.check import CheckResult, check
ModuleNotFoundError: No module named 'tach.check'
```

Probably I can help fix it but I need some guidance :-D

**Comment by cameronbrill** [comment]
looks like `tach.check` [was removed](https://github.com/gauge-sh/tach/commit/01a01c3cf142f480f81d05cf42d56ab488acf53c) cc @sparshg @emdoyle  I can't get the vs code extension working for my org and have the same error message as @algorithmx

**Comment by emdoyle** [comment]
Apologies for the issues here! We will ship an up-to-date release ASAP and ping back here when it is available.

**Comment by caelean** [comment]
@cameronbrill @algorithmx this should be fixed now with v0.14.3! Let me know if you run into any issues.
